vagrant_provision.sh

```diff
@@ -52,9 +52,21 @@ cat > /home/vagrant/hadoop3/etc/hadoop/hdfs-site.xml <<EOF
     </property>
 </configuration>
 EOF
+
+# add custom config files to Hadoop
+cat /home/vagrant/rdfs/config/hdfs-site.xml > /home/vagrant/hadoop/etc/hadoop/hdfs-site.xml
+cat /home/vagrant/rdfs/config/core-site.xml > /home/vagrant/hadoop/etc/hadoop/core-site.xml
```

Comment:
they are the same file, `hadoop` is just a link to `hadoop3` because of `ln -s hadoop3 hadoop` a few lines above.